ezfio.py

```diff
@@ -257,6 +257,18 @@ def CollectDriveInfo():
     # These are nice to have, but we can run without it
     model = "UNKNOWN"
     serial = "UNKNOWN"
+    try:
+        nvmecli = ['nvme', 'list', physDrive]
+        code, info, err = Run(nvmecli)
+        lines = info.split("\n")
+        if len(lines) == 4:
+            data = lines[2].split("  ")
+            model=data[5].lstrip().rstrip()
+            serial=data[2].lstrip().rstrip()
+        else:
```

Comment:
Thanks for the contribution!

This addition seems fine, but there are a couple problems:
* It prints an error message if nvmecli isn't installed even if sdparm is successful (say, testing a SAS or SATA SSD).
* It should return if the nvmecli succeeds, otherwise it will fall thru and try sdparm which may pass (in which case it will throw out the nvmecli results) or fail and print a message saying that sdparm needs to be installed (which isn't true).

The best way for this would be to return on success from either attempt and no print a message until after both fail, saying something like "if it's an nvme drive, install ncme-cli.  OTW install sdparm."